packages/mobile/test/schemas.ts

```diff
@@ -9,6 +9,7 @@ export const vNeg1Schema = {
     numberVerified: false,
     error: null,
     dismissErrorAfter: null,
+    doingPinVerification: false,
```

Comment:
This should go in the latest schema version (v3). The old ones cannot be modified.